.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Config file for credentials
+config.py
```

Comment:
Could you also provide a template of the config (and update README explaining that it needs to be populated).

Name could be `config.template.py`

Reply:
Added config template file to input API credentials and updated readme.